ArduSub/Sub.h

```diff
@@ -233,7 +235,6 @@ class Sub : public AP_HAL::HAL::Callbacks {
         struct {
             uint8_t unused1             : 1; // 0
             uint8_t simple_mode         : 2; // 1,2     // This is the state of simple mode : 0 = disabled ; 1 = SIMPLE ; 2 = SUPERSIMPLE
-            uint8_t pre_arm_rc_check    : 1; // 3       // true if rc input pre-arm checks have been completed successfully
```

Comment:
if you remove this then all your index change. This either needs to be renamed to "unused1" or all the comments below it need to shift up one. Also, does this shift effect anything? Anyone compare bits indexes in .value?